src/applications/mhv-secure-messaging/containers/AuthorizedRoutes.jsx

```diff
@@ -74,6 +79,11 @@ const AuthorizedRoutes = () => {
         >
           <FolderThreadListView />
         </AppRoute>
+        {contactListPage && (
+          <AppRoute exact path={Paths.CONTACT_LIST} key="ContactList">

```

Comment:
this is not impactful, but for consistency, we may want to keep `key` values the same as of the child component i.e. `EditContactList`. But again, it is not criticial and does not impact routing in any way